spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -28,6 +28,36 @@ This section describes the REST-api, that monitoring agents would use to retriev
 A sub-tree that does not contain data must be omitted.
 * A 'shadow tree' that responds to OPTIONS will provide the metadata.
 
+==== Translation rules for metric names
+
+The metric name in JSON is comprised of the metric name appended with the metric tags (if they exist).
+The metric tags is represented by a comma separated list of key/value tag pairs surrounded by braces `{` and `}`
```

Comment:
Should we say that the tags should be in alphabetical order of their keys? Are duplicate keys allowed?